packages/lodestar-cli/src/cmds/account/cmds/wallet/create.ts

```diff
@@ -116,12 +95,9 @@ wallets are supported presently.",
   recover your private keys in the case of data loss. Writing it on 
   a piece of paper and storing it in a safe place would be prudent.
 
-  Your wallet's UUID is:
-
-  \t${uuid}
-
-  You do not need to backup your UUID or keep it secret.
-  `);
+  `,
+      printUuidData(uuid)
```

Comment:
This util is not necessary and may mess up the print style if tabulations are not computed properly. No problem in duplicating the output in two commands